### PR TITLE
add extra version in header if extra version is not stable

### PR DESCRIPTION
**Pull request by SOLenG**

### CODE REVIEW

File: BackofficeBundle/Resources/public/less/template.less [hunk 1]
```diff
@@ -15,13 +15,20 @@ ul, li {
...
-  color: #BFBFBF;
-  font-size: 13px;
-  font-weight: bold;
-  text-transform: uppercase;
-  text-shadow: 0 0 1px white;
+.no-body-no-break {
+  width: 0;
+  white-space: nowrap;
+
+  #oo-version {
```
Comment (nicolasThal) [inline BackofficeBundle/Resources/public/less/template.less]: do not put the style on an id, it is less reusable

**Comment by Thiblef** [comment]
:+1:

File: BackofficeBundle/Resources/public/less/smartadminPatches/logo.less [hunk 2]
```diff
@@ -2,6 +2,7 @@
   width: 150px;
 }
 #logo {
+  width: 0;
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/public/less/smartadminPatches/logo.less]: why width 0 ? it's weird

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**